File: ImageViewer/Views/MainWindow.axaml.fs
namespace ImageViewer.Views

open Avalonia
open Avalonia.Controls
open Avalonia.Markup.Xaml
open ImageViewer.ViewModels

type MainWindow () as this = 
    inherit Window ()

    do this.InitializeComponent()


    member private this.InitializeComponent() =
#if DEBUG
        this.AttachDevTools()
#endif
        AvaloniaXamlLoader.Load(this)

File: Clock/Views/MainWindow.axaml.cs
using Avalonia;
using Avalonia.Controls;
using Avalonia.Markup.Xaml;
using System;

namespace Clock.Views
{
    public partial class MainWindow : Window
    {
        public MainWindow()
        {
            InitializeComponent();
#if DEBUG
            this.AttachDevTools();
#endif
        }

        private void InitializeComponent()
        {
            AvaloniaXamlLoader.Load(this);
        }
    }
}
